doc/release/RELEASE-NOTES.md

```diff
@@ -24,15 +24,15 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 * *Describe architecture/environment changes here.*
 
 ### Bugfix
-* *Describe bugfix here.*
-
+* [OSDEV-576](https://opensupplyhub.atlassian.net/browse/OSDEV-576) - Implemented one source of truth to Search query source & Production Location Details page source for field `number_of_workers`.
 ### What's new
 * [OSDEV-1090](https://opensupplyhub.atlassian.net/browse/OSDEV-1090) - Claims. Remove extra product type field on Claimed Facility Details page.
 * [OSDEV-273](https://opensupplyhub.atlassian.net/browse/OSDEV-273) - Facility Claims. Implement filtering by Country and Status. Set 'pending' claim status as a default filter.
 * [OSDEV-1083](https://opensupplyhub.atlassian.net/browse/OSDEV-1083) - Implemented a 'toggle password visibility' feature in the login, registration, reset password and user profile forms.
 
 ### Release instructions:
-* *Provide release instructions here.*
+* Apply DB migrations up to the latest one. (post_deploy)
+* Run the `index_facilities_new` management command. (post_deploy)
```

Comment:
Should we add running of this command to the `post_deployment` command?
cc @mazursasha1990

Reply:
@vladsha-dev Yes, we need to, I can do it on current PR if needed.
cc @mazursasha1990

Reply:
The `post_deployment` command already includes a command that runs migrations. 
I believe it's okay to include the `index_facilities_new` command in the `post_deployment` command in the current PR.

In my opinion, it would be better to add the following sentence to the 'Release instructions':
Ensure that the following commands are included in the `post_deployment` command:

- migrate
- index_facilities_new

Reply:
@mazursasha1990 updated